### DISCUSSION TITLE
Spatial Awareness for mobile devices

### DISCUSSION THREAD

**Discussion by RibinMathewChalumattu** [microsoft/MixedRealityToolkit-Unity#9417] [Q&A] [answered]
Dear devs,

Does spatial awareness work with android and ios devices? For example, could the depth information of the LiDAR sensor from an iPhone be utilized, similar to how the hololens depth data is used? If no, can we expect support for spatial awareness anytime soon?

Best,

Ribin

**Comment by david-c-kline** [comment, marked as answer]
At this time, the Spatial Awareness feature works only on HoloLens devices. It is possible, and is on our roadmap, to implement a data provider that supports Unity's AR Foundation, which would add support for detecting planes on Android and iOS.
